devtools/app/components/ComponentPreview.vue

```diff
@@ -0,0 +1,125 @@
+<script setup lang="ts">
+import type { Component } from '../../../src/devtools/rpc'
+import { useClipboard } from '@vueuse/core'
+import { kebabCase } from 'scule'
+import { genObjectFromValues, genArrayFromRaw } from 'knitwork'
+
+const props = defineProps<{ component?: Component, props?: object, themeSlots?: Record<string, any> }>()
+
+const { data: componentExample } = await useAsyncData('__ui_devtools_component-source', async () => {
+  const example = props.component?.meta?.devtools?.example
+  if (!example) return false
+  return await $fetch<{ source: string }>(`/api/component-example`, { params: { component: example } })
+}, { watch: [() => props.component?.slug] })
+
+const code = computed(() => {
+  if (!props.component) return
+
+  const propsTemplate = Object.entries(props.props ?? {}).map(([key, value]: [string, any]) => {
+    if (value === true) return kebabCase(key)
+    if (value === false) return
+    if (!value) return
+    if (props.component?.defaultVariants?.[key] === value) return
+    if (props.component?.meta?.props.find(prop => prop.name === key && prop.default === value)) return
```

Comment:
There's an issue here leading to some default (multi-words) props  behind displayed in the preview most likely caused by diffs in casing.